File: app/src/main/java/com/akay/fitnass/data/db/dao/ActiveRunsDao.java
package com.akay.fitnass.data.db.dao;

import android.arch.lifecycle.LiveData;
import android.arch.persistence.room.Dao;
import android.arch.persistence.room.Query;

import com.akay.fitnass.data.model.ActiveRuns;

@Dao
public interface ActiveRunsDao extends BaseDao<ActiveRuns> {

    @Query("SELECT * FROM active_runs WHERE id = 1")
    ActiveRuns getActiveRuns();

    @Query("SELECT * FROM active_runs WHERE id = " + ActiveRuns.ID)
    LiveData<ActiveRuns> getLiveActiveRuns();

    @Query("DELETE FROM active_runs")
    void deleteActiveRuns();
}

File: app/src/main/java/com/akay/fitnass/data/db/dao/ActiveRunsDao.kt
package com.akay.fitnass.data.db.dao

import androidx.lifecycle.LiveData
import androidx.room.Dao
import androidx.room.Query

import com.akay.fitnass.data.model.ActiveRuns

@Dao
interface ActiveRunsDao : BaseDao<ActiveRuns> {

    @Query("SELECT * FROM active_runs WHERE id = 1")
    fun getActiveRuns(): ActiveRuns

    @Query("SELECT * FROM active_runs WHERE id = 1")
    fun getLiveActiveRuns(): LiveData<ActiveRuns>

    @Query("DELETE FROM active_runs")
    fun deleteActiveRuns()
}
